spec/classes/manage_spec.rb

```diff
@@ -38,6 +39,6 @@
 
     it { is_expected.to compile }
     it { is_expected.to contain_file('/etc/motd.d/hello').with_content('I say Hi').with_notify('Service[sshd]') }
-    it { is_expected.to contain_package('example').with_ensure('installed') }
+    it { is_expected.to contain_package('example').with_ensure('installed').with_subscribe(['Service[sshd]', 'File[/etc/motd.d]']) }
```

Comment:
It is better to use `that_subscribes_to` instead of `with_subscribe`. The former checks the relation exists in the catalog, the latter checks whether the attribute is present.
```suggestion
    it { is_expected.to contain_package('example').with_ensure('installed').that_subscribes_to(['Service[sshd]', 'File[/etc/motd.d]']) }
```